sensor/upgrader/upgradectx/testutils.go

```diff
@@ -0,0 +1,38 @@
+package upgradectx
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/clientconn"
+	"github.com/stackrox/rox/pkg/mtls"
+	"github.com/stackrox/rox/sensor/upgrader/config"
+)
+
+// CreateForTest creates a test upgrader context from the given config.
+func CreateForTest(
```

Comment:
This is exported function in `upgradectx` that is used only in test in the `bundle` pkg 🤔 
Couldn't we add this as a helper function in the `bundle` pkg and from there call the constructor of `UpgradeContext`?